docker/create-build-nilrt.sh

```diff
@@ -45,13 +59,15 @@ pushd "${SCRIPT_ROOT}"
 short_hash=$(git rev-parse --short HEAD)
 popd
 
-# parse the NIRLT codename from the meta-nilrt:layer.conf
-NILRT_codename=$(grep -e '^LAYERSERIES_COMPAT_meta-nilrt' "${SCRIPT_ROOT}/../sources/meta-nilrt/conf/layer.conf" | cut -d'"' -f2)
-if [ -z "$NILRT_codename" ]; then
-	echo "ERROR: could not parse NILRT_codename from the meta-nilrt layer." >&2
-	exit 1
-else
-	echo "INFO: using NILRT_codename=${NILRT_codename}"
+# optionally parse the NIRLT codename from the meta-nilrt:layer.conf
+if [ -z "${tag_codename}" ]; then
+	NILRT_codename=$(grep -e '^LAYERSERIES_COMPAT_meta-nilrt' "${SCRIPT_ROOT}/../sources/meta-nilrt/conf/layer.conf" | cut -d'"' -f2)
+	if [ -z "$NILRT_codename" ]; then
+		echo "ERROR: could not parse NILRT_codename from the meta-nilrt layer." >&2
+		exit 1
+	else
+		echo "INFO: using NILRT_codename=${NILRT_codename}"
+	fi
 fi
```

Comment:
Do we need an else condition here that sets `NILRT_codename` to `tag_codename`, or just rename `tag_codename` to `NILRT_codename`? I don't see `NILRT_codename` being initialized anywhere.

Reply:
Ah, dangit. You're right.